octobot_trading/personal_data/orders/wrapped_order.py

```diff
@@ -0,0 +1,61 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_trading.errors as errors
+
+
+class WrappedOrder:
```

Comment:
I would prefer something like `PendingOrder` if this is only for potential future order

Reply:
I also considered `PendingOrder` when naming this, however the issue with `PendingOrder` is that we could think it's an order that is waiting to be published or something like this (like if "pending" was a state of the order). On the other hand, a wrapped order is actually an "order that is waiting to be published". Therefore if you think it's better to name it  `PendingOrder` I'm ok with it